dojo/fixtures/defect_dojo_sample_data.json

```diff
@@ -8620,6 +8620,16 @@
     }
     },
     {
+    "model": "dojo.test_type",
+    "pk": 149,
+    "fields": {
+        "name": "JFrog Xray Vulnerabilities",

```

Comment:
Change  to Jfrog Xray On-Demand Binary Scan

Reply:
rename